packages/tailwindcss-x/demo/options.ts

```diff
@@ -0,0 +1,71 @@
+import { StyleOptions, DynamicStylesOptions, PluginOptions } from '../src/types';
+
+export const options: PluginOptions = {
+  components({ theme }): StyleOptions {
+    return {
+      '.icon': {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        width: theme('spacing.16'),
+        height: theme('spacing.16'),
+        color: 'currentColor',
+        fontSize: theme('spacing.8')
+      }
```

Comment:
The types are not working. Here we should have autocomplete for overriding default components and is not working.

We should have this:
<img width="551" alt="image" src="https://user-images.githubusercontent.com/5759712/163967306-5f2387a6-aa84-4ea6-83b1-42c3be9e98de.png">

The same for utilities and dynamic components/utilities

Reply:
Fixed